src/components/ShareTrajectoryModal/index.tsx

```diff
@@ -146,148 +136,6 @@ const ShareTrajectoryModal = ({
-            <div className={styles.embedHeader}>
-                <h4>Embed &lt;/&gt;</h4>
-                <button
-                    className={styles.settingsButton}
-                    onClick={() => {
-                        setShowEmbedSettingsPanel(!showEmbedSettingsPanel);
-                    }}
-                >
-                    Show settings {DownArrow}
-                </button>
-            </div>
-            {showEmbedSettingsPanel && (
-                <div className={styles.embedSettings}>
-                    <VerticalFlexbox gap={4}>
-                        <p className={styles.accentText}> Size </p>
-                        <VerticalFlexbox gap={24}>
-                            <div className={styles.proportionSettings}>
-                                {" "}
-                                <Input
-                                    disabled={
-                                        !embedSettings.userProvidedProportions
-                                    }
-                                    defaultValue={embedSettings.height}
-                                    className={styles.numberInputs}
-                                    onChange={(e) => {
-                                        handleEmbeddedSizeInputs(
-                                            "height",
-                                            e.target.value
-                                        );
-                                    }}
-                                />
-                                <p> x </p>
-                                <Input
-                                    disabled={
-                                        !embedSettings.userProvidedProportions
-                                    }
-                                    value={embedSettings.width}
-                                    className={styles.numberInputs}
-                                    onChange={(e) => {
-                                        handleEmbeddedSizeInputs(
-                                            "width",
-                                            e.target.value
-                                        );
-                                    }}
-                                />
-                                <p> pixels </p>
-                                <div className={styles.constrainProportions}>
-                                    <Checkbox
-                                        onChange={() => {
-                                            setEmbedSettings({
-                                                ...embedSettings,
-                                                ["userProvidedProportions"]:
-                                                    !embedSettings.userProvidedProportions,
-                                            });
-                                        }}
-                                    ></Checkbox>
-                                    <p>Constrain proportions</p>
```

Comment:
if you wrap the checkbox in a label, then you can click the constrain and it will uncheck/check the box